packages/ui/src/components/va-avatar-group/VaAvatarGroup.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div
+    class="va-avatar-group"
+    :class="classComputed"
+    role="group"
+  >
+    <va-avatar
+      v-for="(option, idx) in maxOptions"
```

Comment:
If we have `role=group`, here we need an `listitem` or `treeitem` role (i'd prefer `listitem`).